znai-docs/znai/snippets/python.md

```diff
@@ -1,8 +1,8 @@
-# Function Content
+# Content
 
 Note: Function Content support requires running znai in an environemnt with Python 3.8 or later.
 
-Use `include-python` plugin to extract a function content.
+Use `include-python` plugin to extract functuion, class or global variable content.
```

Comment:
“Function” misprint. 
could you please add the actual example of extracting global variable.

Reply:
Doh.